test/orm/declarative/test_typed_mapping.py

```diff
@@ -2171,12 +2171,20 @@ class A(Base):
             data: str = Column(String)
             bs: List["B"] = relationship("B", back_populates="a")  # noqa: F821
 
+        class C(Base):
```

Comment:
so i was hoping to do this test like this:

```py
@testing.variation("collection_type", [("list", testing.requires.py310), "List", ("set", testing.requires.py310), "Set"])
def test_14_style_anno_accepted_w_allow_unmapped(self, collection_type):


       class A(Base):
            __tablename__ = "a"
            id: Mapped[int] = mapped_column(primary_key=True)
            data: str = Column(String)

            if collection_type.list:
                bs: list["B"] = relationship("B", back_populates="a")  # noqa: F821
            elif collection_type.List:
                bs: List["B"] = relationship("B", back_populates="a")  # noqa: F821
            elif collection_type.set:
                bs: set["B"] = relationship("B", back_populates="a")  # noqa: F821
            elif collection_type.Set:
                bs: Set["B"] = relationship("B", back_populates="a")  # noqa: F821
            else:
                  collection_type.fail()
```

that way the test itself tests just one thing at a time, we can block older Python versions where `list`, `set` are not subscriptable without losing the tests for `List`, `Set`, the actual test case stays simpler.

Reply:
the `requires.py310` needs to be something like `requires.python310`

Reply:
Thanks for spelling out the solution. I've used testing.combinations instead of testing.variation.